script.js

```diff
@@ -89,3 +89,275 @@ form.addEventListener("submit", (event) => {
+    liveLink: 'https://www.google.com',
+    githubLink: 'https://www.github.com',
+    technologies: [
+      'Codekit',
+      'GitHub',
+      'JavaScript',
+      'Bootstrap',
+      'Sass',
+      'CodePen',
+    ],
+    tech: ['HTML', 'CSS', 'JavaScript', 'Ruby on Rails'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    project_Image: './assets/Snapshoot Portfolio.png',
+    desktop: './assets/Snapshoot Portfolio1.png',
+    mobile: './assets/Snapshoot Portfolio1-mobile.png',
+  },
+  {
+    featuredImage: './assets/Snapshoot Portfolio 6.png',
+    title: 'Keeping track of hundreds of components',
+    liveLink: 'https://www.google.com',
+    githubLink: 'https://www.github.com',
+    technologies: [
+      'Codekit',
+      'GitHub',
+      'JavaScript',
+      'Bootstrap',
+      'Sass',
+      'CodePen',
+    ],
+    tech: ['HTML', 'CSS', 'JavaScript', 'Ruby on Rails'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    project_Image: './assets/Snapshoot Portfolio.png',
+    desktop: './assets/Snapshoot Portfolio1.png',
+    mobile: './assets/Snapshoot Portfolio1-mobile.png',
+  },
+];
+
+const projectSection = document.getElementById('projects');
+projectSection.className = 'works-samples-container';
+
+const projectsHeading = document.createElement('h2');
+projectsHeading.className = 'works-samples-heading';
+projectsHeading.textContent = 'My Recent Works';
+projectSection.appendChild(projectsHeading);
+
+function closePopup(popup) {
+  document.body.removeChild(popup);
+}
+
+function showProjectDetails(project) {
+  const popup = document.createElement('div');
+  popup.className = 'popup';
+
+  const popupWindow = document.createElement('section');
+  popupWindow.className = 'popup-window';
+
+  const closeButton = document.createElement('button');
+  closeButton.className = 'close-popup-button';
+  closeButton.textContent = 'X';
+  closeButton.addEventListener('click', () => {
+    closePopup(popup);
+  });
+  popupWindow.appendChild(closeButton);
+
+  const projectImage = document.createElement('img');
```

Comment:
Good job implementing some of the changes requested by the previous code reviewer but ,As suggested by the previous code reviewer the desktop and mobile design popup window is not exactly the design we have for the template in your figma design , please ensure to use the right format , carefully inspect the template and always pay attention to detail:

This is is your desktop design:

![Screenshot 2023-05-20 at 12 43 51 PM](https://github.com/Peter-Kibet/Peter-K-Portfolio/assets/79506105/96ae4777-510c-4fd1-9f37-b68ef07fd3b4)

This is your mobile design:

![Screenshot 2023-05-20 at 12 43 41 PM](https://github.com/Peter-Kibet/Peter-K-Portfolio/assets/79506105/4cc397f4-9b16-4c48-a3f9-c4746a8e0913)

Please follow this FIGMA design:

Desktop view should look similar to this:

![239666066-c1dd9d5b-d64e-4d27-8512-ddd722efac3c](https://github.com/Peter-Kibet/Peter-K-Portfolio/assets/79506105/33c6702d-bd60-45f7-a908-ae83339e4016)

Your mobile view should look like this:

![239666188-579b3d49-3dbd-40c3-a1a4-38da6df39ad8](https://github.com/Peter-Kibet/Peter-K-Portfolio/assets/79506105/54fccf5e-2dbf-4c21-8e92-b10908f90276)

It would be nice to make these correction always pay attention to detail to avoid making mistakes.